README.md

```diff
@@ -155,6 +155,17 @@ values of JSONPath expressions that evaluate to arrays/slices of numbers.
 It's optional but when the expression evaluates to an array/slice, it's absence will
 produce an error. The supported aggregation functions are `avg`, `max`, `min` and `sum`.
 
+The `rawQuery` configuration option specifies the query params to send along to the endpoint:
+```yaml
+  metric-config.pods.requests-per-second.json-path/path: /metrics
+  metric-config.pods.requests-per-second.json-path/port: "9090"
+  metric-config.pods.requests-per-second.json-path/rawQuery: "foo=bar&baz=bop"
```

Comment:
For all other keys we use a `-` as separator, so maybe it's better to call it `raw-query` to be consistent with that.